aviary/modules/binning/scripts/run_checkm.py

```diff
@@ -0,0 +1,26 @@
+import subprocess
+import shutil
+import os
+from pathlib import Path
+
+def checkm(checkm2_db, bin_folder, bin_ext, refinery_max_iterations, output_folder, output_file, threads):
+    if len([f for f in os.listdir(bin_folder) if f.endswith(bin_ext)]) > 0 and refinery_max_iterations > 0:
+        print(f"Using CheckM2 database {checkm2_db}/uniref100.KO.1.dmnd")
+        subprocess.run(f"CHECKM2DB={checkm2_db}/uniref100.KO.1.dmnd checkm2 predict -i {bin_folder}/ -x {bin_ext} -o {output_folder} -t {threads} --force")
+        shutil.copy(f"{output_folder}/quality_report.tsv", output_file)
+    else:
+        print(f"No bins found in {bin_folder}")
+        os.makedirs(output_folder)
+        Path(output_file).touch()

```

Comment:
Maybe a less scary message should be printed when `refinery_max_iterations==0` so users don't get confused